crates/vm/levm/src/opcode_handlers/system.rs

```diff
@@ -173,10 +173,8 @@ impl VM {
 
         let new_memory_size = calculate_memory_size(offset, size)?;
 
-        let memory_expansion_cost: u64 =
-            memory::expansion_cost(new_memory_size, current_call_frame.memory.len())?
-                .try_into()
-                .map_err(|_err| VMError::Internal(InternalError::ConversionError))?;
```

Comment:
Ditto. This furthers the "it slipped" theory. It's the same function so it's strange that a different error is returned.